src/status_im/translations/en.cljs

```diff
@@ -178,7 +178,7 @@
    :wallet-set-up-signing-phrase         "This is your personal transaction phrase that you’ll use everytime you make a transaction. Make sure to write it down on a piece of paper, store it somewhere, and only confirm transactions when you see these three words."
    :wallet-set-up-confirm-title          "Wrote it down?"
    :wallet-set-up-confirm-description    "You won’t be able to see your 3-word transaction phrase again after this."
-   :wallet-backup-recovery-title         "Backup your Recovery phrase"
+   :wallet-backup-recovery-title         "Backup your recovery phrase"
    :wallet-backup-recovery-description   "This will help you to keep your money safe"
```

Comment:
We don't store money. Should we say assets instead?

Reply:
Yes, good catch!